src/main/java/eu/dissco/core/translator/terms/media/MediaHost.java

```diff
@@ -0,0 +1,12 @@
+package eu.dissco.core.translator.terms.media;
+
+import eu.dissco.core.translator.terms.specimen.OrganisationId;
+
+public class MediaHost extends OrganisationId {
+  public static final String TERM = "mediaHost";
```

Comment:
We will be switching this anyway. But I would say we add the organisationId and organisationName to the digital media object instead of a new term.